content/ko/docs/contribute/localization_ko.md

```diff
@@ -328,16 +361,20 @@ skew | 차이(skew) |
 spec | 명세, 스펙, 사양 |
 specification | 명세 |
 Stateful Set | 스테이트풀 셋 |
+StatefulSet | 스테이트풀셋(StatefulSet) |
 stateless | 스테이트리스 |
 Static pod | 스태틱(static) 파드 |
 Storage Class | 스토리지 클래스 |
```

Comment:
Maybe remove “Storage Class” in favor of “Storage Class”? Or lowercase the English column for this row to “storage class”.

Reply:
```suggestion
storage class | 스토리지 클래스 |
StorageClass | 스토리지클래스(StorageClass) |
```
You mean like this?